packages/icons/src/material/account_circle.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
So what happened to the "no dimensions inside svg" convention? Is it not applicable to design system?